README.md

```diff
@@ -250,28 +260,131 @@ kpt alpha repo register \
+
+In fact, there is no reason you can't just change the resources as you wish, directly.
+The Configuration as Data approach is open to the mindset of "you can edit the config either via code or manually". 
+
+Example:
+```
+kpt pkg get --for-deployment https://github.com/nephio-project/nephio-packages.git/nephio-configsync nephio-edge-cluster-01
+# Replace the repository name in rootsync.yaml with the $GITHUB_USERNAME
+kpt fn eval nephio-edge-cluster-01 \
+  --save \
+  --type mutator \
+  --image gcr.io/kpt-fn/search-replace:v0.2.0 \
+  -- by-path=spec.git.repo by-value-regex='https://github.com/[a-zA-Z0-9-]+/(.*)' \
+  put-value="https://github.com/${GITHUB_USERNAME}/\${1}"
+kpt fn render nephio-edge-cluster-01
+kpt live init nephio-edge-cluster-01
+kpt live apply nephio-edge-cluster-01 --reconcile-timeout=15m --output=table
+
+# Verify that the config sync pods are running
+kubectl get pods --all-namespaces | grep config
+config-management-monitoring   otel-collector-5b8d5d8d8f-8qs2k                                 1/1     Running   0          1m
+config-management-system       config-management-operator-6d7867f9f5-xb9cn                     1/1     Running   0          3m
+config-management-system       reconciler-manager-c6c8cf7f6-9x7vr                              2/2     Running   0          1m
+config-management-system       root-reconciler-nephio-workload-cluster-sync-679dd89788-mkl8x   4/4     Running   0          1m
+```
+
+## Deploying a Package Workload
+We have two options to provision a sample workload.
+  * Use the Nephio Web UI in case you followed the steps in [Installing the Nephio Web UI](#installing-the-web-ui) above, or
+  * Use `kpt` to provision the workload from the command line.
+
+[Installing the Nephio Web UI](#installing-the-web-ui) (as described above) was optional.
+Therefore, we will use `kpt` to provision the workload.
 
 Example:
 ```
-kpt pkg get --for-deployment https://github.com/nephio-project/nephio-packages.git/nephio-configsync test-deploy-edge-01
-kpt fn render test-deploy-edge-01
-kpt live init test-deploy-edge-01
-kpt live --context=gke_jbelamaric-dev_us-central1_nephio-edge-01 apply test-deploy-edge-01 --reconcile-timeout=15m --output=table
+# Important! We need to set our Kubernetes context back to the nephio cluster first!
+kubectl config get-contexts | grep nephio
+* gke_nephio-poc_us-central1_nephio-edge-cluster-01   gke_nephio-poc_us-central1_nephio-edge-cluster-01   gke_nephio-poc_us-central1_nephio-edge-cluster-01
+  gke_nephio-poc_us-central1_nephio                   gke_nephio-poc_us-central1_nephio                   gke_nephio-poc_us-central1_nephio                       
+
+# Your contexts will be named slightly differently. Make sure you adjust the following command accordingly.
+kubectl config use-context gke_nephio-poc_us-central1_nephio
+
+# Get the remote packages available from the registered repositories (output trunkated to focus on coredns-caching package).
```

Comment:
s/trunkated/truncated/

Reply:
Hehehe, good find :-)
Not everything in Kubeland is spelled with a k
Will fix and send another small PR with the correct spelling.